### ISSUE TITLE
why does the ent_coef grow so big

### ISSUE THREAD

**Issue by war3gu** [DLR-RM/stable-baselines3#781]
i change the super params as below
env_kwargs_train = {
"stock_dim": stock_dimension,
"hmax": 1000,
"initial_amount": 1000000,
"buy_cost_pct": 6.87e-5,
"sell_cost_pct": 1.0687e-3,
"reward_scaling": 1e0,
"state_space": state_space,
"action_space": stock_dimension,
"tech_indicator_list": config.TECHNICAL_INDICATORS_LIST,
"print_verbosity": 100,
"initial_buy": True,
"hundred_each_trade": True,
"out_of_cash_penalty": 0.001,
"cash_limit": 0.2,
"model_name":"stock_a",
"mode":"train",
"random_start":True
}

SAC_PARAMS = {
    "batch_size": 1024,   
    "buffer_size": 100000,
    "learning_rate": 0.0002,
    "action_noise": "normal",
    "gradient_steps": 100,    
    "train_freq": (1000, "step"),  
    "learning_starts": 10,
    "ent_coef": "auto_0.1",
}

POLICY_KWARGS = dict(net_arch=dict(pi=[256, 128, 128, 64, 64], qf=[256, 128, 128, 64, 64]))
the result is as below
| time/ | |
| episodes | 4 |
| fps | 759 |
| time_elapsed | 5 |
| total_timesteps | 4264 |
| train/ | |
| actor_loss | 769 |
| critic_loss | 2.35e+05 |
| ent_coef | 0.101 |
| ent_coef_loss | 64.6 |
| learning_rate | 0.0002 |
| n_updates | 100 |
| reward | 0 |
| time/ | |
| episodes | 16 |
| fps | 767 |
| time_elapsed | 22 |
| total_timesteps | 17284 |
| train/ | |
| actor_loss | 768 |
| critic_loss | 1.47e+05 |
| ent_coef | 0.107 |
| ent_coef_loss | 62.9 |
| learning_rate | 0.0002 |
| n_updates | 400 |
| reward | 23.1 |
| time/ | |
| episodes | 472 |
| fps | 750 |
| time_elapsed | 665 |
| total_timesteps | 499640 |
| train/ | |
| actor_loss | 2.18e+03 |
| critic_loss | 1.11e+05 |
| ent_coef | 1.18 |
| ent_coef_loss | -4.72 |
| learning_rate | 0.0002 |
| n_updates | 12400 |
| reward | 17.5 |
this is the first loop result
the last loop result is as below
| time/ | |
| episodes | 400 |
| fps | 748 |
| time_elapsed | 565 |
| total_timesteps | 423104 |
| train/ | |
| actor_loss | 2.75e+12 |
| critic_loss | 9.97e+22 |
| ent_coef | 4.87e+09 |
| ent_coef_loss | -629 |
| learning_rate | 0.0002 |
| n_updates | 123000 |
| reward | 0 |
everything grow to a super big value
auto_0.1 means that ent_coef start from 0.1
you can get my code from https://github.com/war3gu/FinRL-Meta
run the file Demo_China_A_share_market_sac.py
my python version is 3.7

**Comment by Miffyli** [comment]
We do not offer tech support for getting RL working for task X. Your best bet is to read the docs for tips in addition to other sources of RL tips. Apart from that you need to experiment and see what works.

--- 

**The following is an automated answer:**

as you seem to try to apply RL to stock trading, i also must warn you about it. Here is recommendation from a former professional trader:

Retail trading, retail trading with ML, and retail trading with RL are bad ideas for almost everyone to get involved with. 
* I was a quant trader at a major hedge fund for several years. I am now retired.
* On average, traders lose money. On average, retail traders especially lose money. An excellent approximation of trading, and especially of retail trading, is 'gambling'. 
* There is a lot more bad advice on trading out there than good advice. It is extraordinarily difficult to demonstrate that any particular advice is some of the rare good advice.
* As such, it's reasonable to treat all commentary on retail trading as an epsilon away from snake oil salesmanship. Sometimes that'll be wrong, but it's a strong rule of thumb. 
* I feel a sense of responsibility to the less world-wise members of this community - which includes plenty of highschoolers - and so I find myself unable to let a conversation about retail trading occur without interceding and warning that it's very likely snake oil. 
* I find repeatedly making these warnings and the subsequent fights to be exhausting.
